test/server/index.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright 2015 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+'use strict';
+
+// This server is needed most importantly to add the Service-Worker-Allowed
+// header to any service worker loaded in the tests. This allows the scope
+// to be manipulated any way we want / need during testing.
+
+var path = require('path');
+var express = require('express');
+var app = express();
+
+// Set up static assets
```

Comment:
On the fence with this.

For now, the tests built in such a way that we don't need to account for caching scenarios (i.e. at no stage are the assets be altered, they are always the same, so caching isn't an issue).

If we get to a stage of testing service upgrade paths then caching manipulation becomes important and in that case we'll likely want ways of enabling / disabling caching to ensure the behavior is consistent.

Happy to add it - but I don't feel like it's useful at this stage.